### ISSUE TITLE
A Web Site Live Feed Will Not Start When Using the Web App

### ISSUE THREAD

**Issue by regisstration** [eladnava/applicationize#72]
I created an app that opens the following site, "https://www.livenewson.com/american/msnbc.html". When I browse to the site in Chrome, the live feed starts up on its own. The newly created web app goes to the site, but does not start the feed.

Is there a setting (or otherwise) that can be set to make it behave just like the chrome browser tab?

Thanks

**Comment by eladnava** [comment]
Hi @regisstration,
Apologies for the delayed response.

It seems to work for me, can you try now?

![screen shot 2018-11-14 at 12 06 11 pm](https://user-images.githubusercontent.com/4394189/48459588-bbeaae00-e805-11e8-9995-3cfdc65e9bfa.png)

**Comment by regisstration** [comment]
Thanks for the reply! I tried again with the same result. The stream does not start. I did however see that the video 'thinks' it is streaming! If I hover over the video, I can see that the 'pause' symbol shows in the lower left in the video controls. This implies that it IS streaming, but it's just paused. I confirmed this by clicking on it (the pause symbol), and the .play' symbol appears. Then I click the 'play' symbol, and the stream actually starts! 

I attached the setup, in case I'm missing something. Could it be some other setting in my Chrome that's preventing the stream to 'go' right away? I allow notifications, pop ups and sound for that site.

Thanks again . . .
[app setup page.pdf](https://github.com/eladnava/applicationize/files/2579557/app.setup.page.pdf)
